apps/aecore/src/aec_block_micro_candidate.erl

```diff
@@ -139,6 +136,18 @@ int_create_block(PrevBlockHash, PrevBlock, KeyBlock, Trees, Txs) ->
     BlockInfo = #{ trees => Trees2, txs_tree => TxsTree, tx_env => Env},
     {ok, NewBlock, BlockInfo}.
 
+determine_new_time(PrevBlock) ->
+    LastTime = aec_blocks:time_in_msecs(PrevBlock),
+    case aec_blocks:type(PrevBlock) of
+        key ->
+            %% Just make sure we are progressing time.
+            max(aeu_time:now_in_msecs(), LastTime);
```

Comment:
/me checkng if [`+1`](https://github.com/aeternity/protocol/blame/216819c3629d333ae455c0fb51d13009cfed961d/consensus/consensus.md#L164) needed...

Reply:
Seems +1 needed https://github.com/aeternity/epoch/blob/1128ea24bbf39c4dd71b528c8bac2621ca6edee3/apps/aecore/src/aec_headers.erl#L715

Not surely an issue, though blocking for being sure.